docs/assets/airframes/vtol/omp_hobby_zmo_fpv/omp_hobby_zmo.params

```diff
@@ -2,70 +2,80 @@
 # Vehicle: VTOL
 # Vehicle-Id Component-Id Name Value Type
 
-1	1	BAT1_CAPACITY	4000.0	9
+1	1	ASPD_SCALE_1	1.076	9
```

Comment:
Given that it's so close to 1 and a "too low" reading is safer than a "too high" reading" (risk of stalling), I would remove setting ASPD_SCALE_1.